### DISCUSSION TITLE
Eli Stevens: Deep Learning with PyTorch

### DISCUSSION THREAD

**Discussion by kkmcgg** [NSCC-COGS/Aestheta#159] [Guides]
https://pytorch.org/assets/deep-learning/Deep-Learning-with-PyTorch.pdf

2020 webbook

**Comment by ynandal99** [comment]
Link broken. Here's the new link https://github.com/borninfreedom/DeepLearning/raw/master/Books/Deep-Learning-with-PyTorch.pdf
